AndroidRunner/Plugins/trepn/README_Trepn.md

```diff
@@ -0,0 +1,13 @@
+# Trepn Plugin
+This plugin collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.
```

Comment:
Here you should also mention that the Trepn plugin depends on the Trepn Android app, which is available in this folder. Then, you can move the `com.quicinc.trepn.apk' file from the `examples` folder to here.

Reply:
Added a small sentence that links to the APK's new location